src/envs/pendulum.jl

```diff
@@ -73,7 +74,7 @@ end
 function state(env::Pendulum)
   θ = env.state.θ
   θvel = env.state.θvel
-  Float64[cos(θ), sin(θ), θvel]
+  Float64[θ, θvel]
```

Comment:
I think exporting `sin` and `cos` is intended to match Pendulum-v0 of OpenAI Gym (https://github.com/openai/gym/wiki/Pendulum-v0#observation)

Reply:
Ah, I see. That makes sense (even though it is never used). The problem is that for more advanced algorithms it is kind of a pain to extract the state now, e.g. to feed into a network because of the struct type PendulumState. But I can make another pull request to fix that in a better way.